docker-compose.yaml

```diff
@@ -125,6 +125,7 @@ services:
     configs:
       - source: svix_sql
         target: /docker-entrypoint-initdb.d/svix.sql
+        mode: 0444
```

Comment:
required of svix initialization fails as postgres is run as posgres the file will only have access to root.